src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,18 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: complete
+$(COMPONENTSDIR)/icd11foundation.owl: $(TMPDIR)/icd11foundation_relevant_signature.txt | component-download-icd11foundation.owl
```

Comment:
## Question: Why `xmlns:schema` created?
- [ ] @matentzn or @twhetzel

I noticed that `components/icd11foundation.yml` has this in its namespace: `xmlns:schema="http://id.who.int/icd/schema/">`. I'm curious as to why this namespace was assigned when (i) I didn't instruct `robot` to do so, (ii) it didn't decide to automatically create any other namespaces (e.g why did it create this namespace, but not `xmlns:entity="http://id.who.int/icd/entity/">`?

Reply:
Are you sure that the original file from WHO does not have that NS declaration?

Reply:
Ah, good idea. Yes it does! It's the one that was set to `:`: `Prefix(:=<http://id.who.int/icd/schema/>)`